packages/atlas/src/api/queries/fragments.graphql

```diff
@@ -10,6 +10,7 @@ fragment BasicChannelFields on Channel {
   follows
   rewardAccount
   channelStateBloatBond
+  cumulativeRewardClaimed
```

Comment:
Do we need this on `BasicChannel`? Seems like this should be useful only in contexts where we already have the full channel?